tests/jobs/test_scheduler_job.py

```diff
@@ -495,45 +439,24 @@ def test_find_executable_task_instances_pool(self):
         assert tis[3].key in res_keys
         session.rollback()
 
-    def test_find_executable_task_instances_order_execution_date(self):
+    def test_find_executable_task_instances_order_execution_date(self, dag_maker):
         dag_id_1 = 'SchedulerJobTest.test_find_executable_task_instances_order_execution_date-a'
         dag_id_2 = 'SchedulerJobTest.test_find_executable_task_instances_order_execution_date-b'
         task_id = 'task-a'
-        dag_1 = DAG(dag_id=dag_id_1, start_date=DEFAULT_DATE, max_active_tasks=16)
-        dag_2 = DAG(dag_id=dag_id_2, start_date=DEFAULT_DATE, max_active_tasks=16)
-        dag1_task = DummyOperator(dag=dag_1, task_id=task_id)
-        dag2_task = DummyOperator(dag=dag_2, task_id=task_id)
+        with dag_maker(dag_id=dag_id_1, max_active_tasks=16) as dag_1:
+            dag1_task = DummyOperator(task_id=task_id)
+        dr1 = dag_maker.create_dagrun(execution_date=DEFAULT_DATE + timedelta(hours=1))
```

Comment:
It may not be relevant, but why does the execution_date change here?

Reply:
It's how the dagrun was created before this change,. I decided not to change it. Look at the deleted line 528

Reply:
I think the original setup was trying to test that if the `DEFAULT_DATE + 1h` run is (somehow) scheduled before the `DEFAULT_DATE` one, tasks in the earlier run will still be run first. It’s probably worthwhile to add a docstring or comment describing the test’s setup.